packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/SplitButton.tsx

```diff
@@ -0,0 +1,142 @@
+    fontSize: ({ fontSize }) => (fontSize ? fontSize : undefined),
+    fontStyle: ({ italic }) => (italic ? 'italic' : undefined),
+    fontWeight: ({ bold }) => (bold ? 'bold' : undefined),
+    textDecorationLine: ({ underline, strikeThrough }) => getTextDecorationLineValue(underline, strikeThrough),
+    '&:hover': {
+      backgroundColor: ({ backgroundColor }) =>
+        backgroundColor ? getCSSColor(backgroundColor, theme) : theme.palette.primary.main,
+      color: ({ foregroundColor }) => (foregroundColor ? getCSSColor(foregroundColor, theme) : 'white'),
+      fontSize: ({ fontSize }) => (fontSize ? fontSize : undefined),
+      fontStyle: ({ italic }) => (italic ? 'italic' : undefined),
+      fontWeight: ({ bold }) => (bold ? 'bold' : undefined),
+      textDecorationLine: ({ underline, strikeThrough }) => getTextDecorationLineValue(underline, strikeThrough),
+    },
+  },
+  icon: {
+    marginRight: ({ iconOnly }) => (iconOnly ? theme.spacing(0) : theme.spacing(2)),
+  },
+  selected: {
+    color: theme.palette.primary.main,
+  },
+  propertySectionLabel: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+}));
+
+export const SplitButtonWidget = ({ widget }: SplitButtonWidgetProps) => {
+  const initialState: SplitButtonWidgetState = {
+    buttonLabel: widget.buttonLabel,
+    selected: false,
+  };
+  const [state, setState] = useState<SplitButtonWidgetState>(initialState);
+
+  const props: ButtonStyleProps = {
+    backgroundColor: widget.actions[0].style?.backgroundColor ?? null,
+    foregroundColor: widget.actions[0].style?.foregroundColor ?? null,
+    fontSize: widget.actions[0].style?.fontSize ?? null,
+    italic: widget.actions[0].style?.italic ?? null,
+    bold: widget.actions[0].style?.bold ?? null,
+    underline: widget.actions[0].style?.underline ?? null,
+    strikeThrough: widget.actions[0].style?.strikeThrough ?? null,
+    iconOnly: state.buttonLabel ? false : true,
+  };
+  const classes = useStyles(props);
+  const { selection } = useSelection();
+
+  const ref = useRef<any>(null);
+
+  useEffect(() => {
+    if (ref.current && selection.entries.find((entry) => entry.id === widget.id)) {
+      ref.current.focus();
+      setState((prevState) => {
+        return {
+          ...prevState,
+          selected: true,
+        };
+      });
+    } else {
+      setState((prevState) => {
+        return {
+          ...prevState,
+          selected: false,
+        };
+      });
+    }
+  }, [selection, widget]);
+
+  return (
+    <div>
+      <div className={classes.propertySectionLabel}>
+        <Typography variant="subtitle2" className={state.selected ? classes.selected : ''}>
+          {widget.label}
+        </Typography>
+        {widget.hasHelpText ? <HelpOutlineOutlined color="secondary" style={{ marginLeft: 8, fontSize: 16 }} /> : null}
+      </div>
+      <ButtonGroup
+        variant="contained"
```

Comment:
A click on the SplitButton dropdown menu does nothing here. May be it should show the list of button widgets that have been declared under the SplitButton.